blockstack_client/cli.py

```diff
@@ -52,106 +46,97 @@
 
 from blockstack_client import config
 from blockstack_client.client import session
-from blockstack_client.config import WALLET_PATH, WALLET_PASSWORD_LENGTH, CONFIG_PATH, VERSION, semver_match
+from blockstack_client.config import CONFIG_PATH, VERSION, semver_match
 from blockstack_client.method_parser import parse_methods, build_method_subparsers
 
-
-from blockstack_profiles import resolve_zone_file_to_profile
-from blockstack_profiles import is_profile_in_legacy_format
-
-from pybitcoin import is_b58check_address
-
-from binascii import hexlify
 from wallet import *
-from utils import exit_with_error, pretty_dump, print_result
+from utils import exit_with_error, print_result
 
 log = config.get_logger()
 
+
 def get_methods(prefix, module):
     """
     Get the built-in CLI methods
     """
     methods = []
     for attr in dir(module):
-        if attr.startswith(prefix):
-            method = getattr(module, attr)
-            if callable(method) or hasattr(method, '__call__'):
-                methods.append( method )
+        if not attr.startswith(prefix):
+            continue
+
+        method = getattr(module, attr)
+
+        if callable(method) or hasattr(method, '__call__'):
+            methods.append(method)
 
     return methods
 
 
-def get_plugin_methods( module_name, prefix ):
+def get_plugin_methods(module_name, prefix):
     """
     Load methods from a given module
     Return the list on success
     Return None on error
     """
     try:
-        mod = __import__(module_name)
+        module = __import__(module_name)
     except ImportError:
-        log.error("Failed to import '%s'" % module_name)
-        return None 
+        log.error('Failed to import "{}"'.format(module_name))
+        return None
 
-    return get_methods( prefix, mod )
-     
+    return get_methods(prefix, module)
 
-def get_cli_basic_methods():
+
+def get_cli_advanced_methods():
     """
-    Get the basic built-in CLI methods
+    Get the advanced usage built-in CLI methods
     """
     import blockstack_client.actions as builtin_methods
-    all_methods = get_methods("cli_", builtin_methods)
-    ret = []
-    for m in all_methods:
-        # filter advanced methods 
-        if 'cli_advanced_' not in m.__name__:
-            ret.append(m)
+    return get_methods('cli_advanced_', builtin_methods)
 
-    return ret
 
-
-def get_cli_advanced_methods():
+def get_cli_basic_methods():
     """
-    Get the advanced usage built-in CLI methods
+    Get the basic built-in CLI methods
     """
     import blockstack_client.actions as builtin_methods
-    return get_methods("cli_advanced_", builtin_methods )
+    all_methods = get_methods('cli_', builtin_methods)
+    advanced_methods = get_cli_advanced_methods()
+
+    return list(set(all_methods) - set(advanced_methods))
 
 
-def prompt_args( arginfolist, prompt_func ):
+def prompt_args(args, prompt_func):
```

Comment:
Could we change `args` back to `arginfolist`?  The reason here is that `args` is often used to refer to a `Namespace(...)` instantiation returned by `argparse` (in blockstack_client/actions.py).  The variable name `arginfolist` here is meant to clarify that this is a list of information about the arguments (i.e. generated by blockstack_client/method_parser.py).